File: graphql/graphql01_kickstarter/src/main/java/com/sc/graphql/Application.java
package com.sc.graphql;

import lombok.RequiredArgsConstructor;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@RequiredArgsConstructor
public class Application {
	public static void main(String[] args) {
		SpringApplication.run(Application.class, args);
	}
}

File: kotlin/kotlin_springboot01/src/main/kotlin/com/sc/kotlin/Application.kt
package com.sc.kotlin

import com.sc.kotlin.entity.Employee
import com.sc.kotlin.repository.EmployeeRepository
import org.springframework.boot.CommandLineRunner
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Bean

// After starting up open
// http://localhost:8080/swagger-ui.html
@SpringBootApplication
class Application {

    // Single line function that create function interface object
    // This syntax create function "init"
    // Dependency inject employeeRepository in it
    // Create instance of CommandLineRunner in its run() implementation create employees
    // returns CommandLineRunner as spring bean
    @Bean
    fun init(employeeRepository: EmployeeRepository) = CommandLineRunner {
        employeeRepository.save(Employee(100, "E1"))
        employeeRepository.save(Employee(200, "E2"))
        employeeRepository.save(Employee(300, "E3"))
    }
}

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
